infrastructure/buckets/s3.tf

```diff
@@ -13,6 +13,13 @@ resource "aws_s3_bucket_object" "minimal_function_object" {
   etag = filemd5("${path.module}/../../dist/helloWorldHandler.zip")
 }
 
+resource "aws_s3_bucket_object" "db_sample_function_object" {
+  bucket = var.bucket_name
+  key = "dbSampleHandler.zip"
+  source = "${path.module}/../../dist/dbSampleHandler.zip"
+  etag = filemd5("${path.module}/../../dist/dbSampleHandler.zip")
+}
+
 # ----------------------------------------------------------------------------------------------------------------------
 # CREATE REFERENCE FROM S3 FILE UPLOADED
```

Comment:
Looks like there is a duplicated definition, the bucket name are the same **kpinetwork-backend** and **var.bucket_name**

Reply:
Done 👍